src/common/badblock_freebsd.c

```diff
@@ -32,6 +32,7 @@
 
 /*
  * badblock_freebsd.c - implementation of the FreeBSD bad block API
+ *                      - NOT SUPPORTED
```

Comment:
BTW, I think we should merge badblock_freebsd.c and badblock_windows.c. The implementation looks quite similar ;).